src/main/scala/it/pagopa/interop/tenantprocess/common/readmodel/ReadModelTenantQueries.scala

```diff
@@ -29,7 +29,8 @@ object ReadModelTenantQueries extends ReadModelQuery {
     val filterPipeline: Seq[Bson] = Seq(
       `match`(query),
       lookup(from = "tenants", localField = "data.id", foreignField = "data.attributes.id", as = "tenants"),
-      unwind("$tenants")
+      unwind("$tenants"),
+      `match`(Filters.not(Filters.exists("tenants.data.attributes.revocationTimestamp")))
```

Comment:
is this filtering all tenants that have at least one revoked attribute, despite the type or the fact that it's the one used for the join?

Reply:
Yes you are right